content/episodes/2019-08-26-S01E06-wer-wacht-ubers-web.md

```diff
@@ -0,0 +1,157 @@
+---
+date: 2019-08-26T00:00:00-00:00
+title: "Wer wacht übers Web"
+ogtitle: "Wer wacht übers Web"
+description: "Triff die Menschen, die deinen Social Media Account aufräumen. What? Jetzt das Staffelfinale von aweb hören!"
+ogdescription: "Delete. Ignore. Delete. - Das ist das Vokabular, das unsere #digitale Realität bestimmt. Die neue Folge #awebpodcast erkundet die #Schattenindustrie der #ContentModeration, also die Welt hinter Facebook und Co. Jetzt hören!"
+season: 1
+seasonepisode: 6
+url: /staffel1/folge6/
+embed: "3610044"
```

Comment:
Just noting: this embed code is seemingly correct but for the moment just plays a short trailer, not the episode. We should verify the episode is live (and is the correct one) before publishing this.